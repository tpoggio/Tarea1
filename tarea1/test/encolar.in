esVaciaCola
encolar 15 5.2
esVaciaCola
esVaciaCola
Fin