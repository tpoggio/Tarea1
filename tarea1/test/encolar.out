1>La cola está vacia. 
2>Encolar. 
3>La cola no está vacia. 
4>La cola no está vacia. 
5>Fin.
